css/share.css

```diff
@@ -47,9 +47,9 @@
 }
 
 #dropdown .shareWithConfirm {
-	padding: 11px;
```

Comment:
Looks good! This padding needs to stay though as makes sure the clickable area is large enough. :) If you just modify the background-color it should be fine?

Reply:
changed!
thanks for the feedback!